komga-webui/src/locales/en.json

```diff
@@ -519,33 +519,33 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could access file during analysis",
+    "ERR_1001": "Unsupported media type",
     "ERR_1002": "Encrypted RAR archives are not supported",
     "ERR_1003": "Solid RAR archives are not supported",
     "ERR_1004": "Multi-Volume RAR archives are not supported",
-    "ERR_1005": "Unknown error while analyzing book",
-    "ERR_1006": "Book does not contain any page",
-    "ERR_1007": "Some entries could not be analyzed",
```

Comment:
Contention

Reply:
"Entries" and "pages" are not the same thing. A page is an entry, but an entry is more generic, and could be files that are not pages. By changing this, you change the meaning.